Goal: Unknown

Fixes: Maybe #164179?

Status wrt upstream: Should maybe be forwarded

Note: 

Index: samba-3.0.25a/source/utils/status.c
===================================================================
--- samba-3.0.25a.orig/source/utils/status.c	2007-05-26 07:45:38.952209923 +0200
+++ samba-3.0.25a/source/utils/status.c	2007-05-26 07:46:34.640653571 +0200
@@ -367,6 +367,16 @@
 	if ( show_locks ) {
 		int ret;
 
+		tdb = tdb_open_log(lock_path("locking.tdb"), 0, TDB_DEFAULT, O_RDONLY, 0);
+
+		if (!tdb) {
+			d_printf("%s not initialised\n", lock_path("locking.tdb"));
+			d_printf("This is normal if an SMB client has never connected to your server.\n");
+			exit(0);
+		} else {
+			tdb_close(tdb);
+		}
+
 		if (!locking_init(1)) {
 			d_printf("Can't initialise locking module - exiting\n");
 			exit(1);
